wordpress-mobile/WordPress-iOS
# Sign up coming from Jetpack install flow causes black screen

Fixes #13826 

### To test:
1. Have a self-hosted site that doesn't have Jetpack connected. (This also works if you go to the site, Jetpack > and scroll to the bottom to find the "Disconnect" button and click it.)
2. Log out of the WPiOS app if you are logged in.
3. Log into the app through the self-hosted flow: Login > enter site address text link button
4. After completing the login flow, go to the Notifications tab
5. Follow the Jetpack install flow. Once it is successful, it will ask you to set up Jetpack.
6. The "Set up" Jetpack button should navigate you to the email login screen. (See Figure 1 below)
7. Tap the "Sign up" text link button. Tap each of the 3 buttons (email / google / apple).
**Expected:** each button navigates the user to the (sign up with email / continue with google / continue with apple) screen.

| Before | After |
| --- | --- |
|<img src="https://user-images.githubusercontent.com/1062444/78588655-9e582300-7804-11ea-88a4-f1be19aa55e3.png" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/78588663-a1ebaa00-7804-11ea-939f-d664569536d8.png" width="350" /> |

### Do not merge until:
1. Authenticator PR merged: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/234 ✔️
2. Authenticator 1.12.1 was published to Cocoapods trunk ✔️
3. This branch points to the updated Auth release.  ✔️

PR submission checklist:
